cmd/web_acme.go

```diff
@@ -47,7 +69,7 @@ func runLetsEncrypt(listenAddr, domain, directory, email string, m http.Handler)
 	magic.Issuers = []certmagic.Issuer{myACME}
 
 	// this obtains certificates or renews them if necessary
-	err := magic.ManageSync(graceful.GetManager().HammerContext(), []string{domain})
+	err := magic.ManageSync(graceful.GetManager().HammerContext(), []string{setting.Domain})
```

Comment:
And here ,before it was `[]string{domain}`, is it changed by purpose?

Reply:
Yes, refactored the whole function to make it simpler. Before it was a mix of function parameters and retrieving settings internally in the function. The function is not used anywhere else, so why not switch to the latter for all. I don't see a situation where you would call `runACME` (previously `runLetsEncrypt`) outside the parameters in the settings.